### PR TITLE
This adds the new customer orders sample schema. 

### PR DESCRIPTION
**Pull request by ChrisSaxonOra**
It includes scripts to create and populates the CO user with the following tables:

- stores
- products
- customers
- orders
- order_items

It also creates the following views:

- store_orders
- product_orders
- customer_order_products
- product_reviews

### CODE REVIEW

File: customer_orders/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+git clone https://github.com/oracle/db-sample-schemas.git
+```
+
+or download and extract the ZIP file:
+
+```shell
+unzip db-sample-schemas.zip
+```
+
+The schema directory should be owned by the Oracle Database software owner.
```
Comment (cjbj) [inline customer_orders/README.md]: These demos must be installable on client machines, where there is no 'Oracle Database' owner.

File: customer_orders/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+### 2.2. Change directory
+
+```shell
+cd $HOME/db-sample-schemas/customer_orders
+```
+
+### 2.4. Set the Oracle environment
+
+```shell
+source /usr/local/bin/oraenv
```
Comment (cjbj) [inline customer_orders/README.md]: Do you care about XE11, where the command is `source /u01/app/oracle/product/11.2.0/xe/bin/oracle_env.sh` ?

File: customer_orders/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+### 3.1. Set the Oracle environment
+
+```shell
+source /usr/local/bin/oraenv
+```
+
+### 3.2. Run the schema removal script
+
+```shell
+sqlplus system/systempw@connect_string
```
Comment (cjbj) [inline customer_orders/README.md]: Remove the password from any command line.

File: customer_orders/README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+Use your actual default and temporary tablespace names.  The passwords 
+for the new CO user will be set to the value you specify.
+
+Using a connect string permits connections to non-container databases and 
+pluggable database using the same syntax.
+
+An example of connect strings for databases with services noncdb and pdb:
+```
+  localhost:1521/noncdb
+  localhost:1521/pdb
```
Comment (cjbj) [inline customer_orders/README.md]: Why not use current real servicenames, e.g. 19c's orclpdb1 ?

File: customer_orders/README.md (hunk 1 above)
Comment (ChrisSaxonOra) [reply customer_orders/README.md]: Just copied from the original readme; this isn't necessary so I've removed it

File: customer_orders/README.md (hunk 2 above)
Comment (ChrisSaxonOra) [reply customer_orders/README.md]: The scripts contain several 12c features, so won't work on 11g! So no.

File: customer_orders/README.md (hunk 4 above)
Comment (ChrisSaxonOra) [reply customer_orders/README.md]: Is there a standard service that's available 12c -> 19c?

File: customer_orders/README.md (hunk 1 above)
Comment (krisrice) [reply customer_orders/README.md]: To add to @cjbj .. No  /usr/local/bin/oraenv

File: customer_orders/README.md (hunk 2 above)
Comment (krisrice) [reply customer_orders/README.md]: Most client machines will not have this:  /usr/local/bin/oraenv needed?

Comment (ChrisSaxonOra) [reply customer_orders/README.md]: Good point. How about just confirming that SQL*Plus is in your path?

**Pull request merged**